Do not show deleted pms in the pm popup

BrickOzp opened the pull request:
The pm popup listed all unread pms, even if they where
deleted. Changed so that deleted pms don't show up.

Fixes #6994

Signed-off-by: Oscar Rydhé <oscar.rydhe@gmail.com>

jdarwood007 commented:
What about the fact that they still show up?  Shouldn't that be corrected as well?

BrickOzp committed:
Do not show deleted pms in the pm popup

The pm popup listed all unread pms, even if they where
deleted. Changed so that deleted pms don't show up.

Fixes #6994

Signed-off-by: Oscar Rydhé <oscar.rydhe@gmail.com>

BrickOzp commented:
> What about the fact that they still show up? Shouldn't that be corrected as well?

New solution should fix that
